### ISSUE TITLE
Exception after upgrade to 1.14.5

### ISSUE THREAD

**Issue by seagullua** [devtodev-analytics/android-sdk#4]
We previously used SDK 1.13.10 and it worked without any issues. The exception appeared after upgrade to 1.14.5.

We do not know how to reproduce the exception, however it happens in the wild in the big chunk of the users. Affected versions of Android include 7, 8, 9, 10

The exception happens in setUserData method.

**Full stack:** 
```
java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.Object java.util.HashMap.get(java.lang.Object)' on a null object reference
com.devtodev.core.logic.b.a(NetworkStorage.java:62)
com.devtodev.core.logic.people.a.c(PlayerPreferences.java:22)
com.devtodev.core.logic.people.a.c(PlayerPreferences.java:6)
com.devtodev.core.logic.people.People$e.run(People.java:2)
com.devtodev.core.logic.f.b.a(EventsQueue.java:10)
com.devtodev.core.logic.SDKClient.addToEventQueue(SDKClient.java:1)
com.devtodev.core.logic.people.People.a(People.java:1)
com.devtodev.core.logic.people.People.setUserData(People.java:4)
com.devtodev.core.logic.people.People.setUserData(People.java:3)
```

**Comment by seagullua** [comment]
We have managed to reproduce the crash.

1. Launch app with SDK 1.13.10. Make sure that DevToDev saved some data to storage.
2. Upgrade the app with APK which contains SDK 1.14.5. It will crash on launch

**Comment by RinatTalibullin** [comment]
Hello!
I'm sorry for inconveniences. Fix will be available ASAP.

**Comment by RinatTalibullin** [comment]
Hello!
SDK was released to version 1.14.6 with issue fixed.
